staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/generated.proto

```diff
@@ -224,7 +224,7 @@ message CustomResourceDefinitionSpec {
   optional CustomResourceDefinitionNames names = 3;
 
   // scope indicates whether the defined custom resource is cluster- or namespace-scoped.
-  // Allowed values are `Cluster` and `Namespaced`. Default is `Namespaced`.
```

Comment:
revert this change, this default is accurate in v1beta1

Reply:
done, thanks :)